docs/examples/django/pages/tests.py

```diff
@@ -0,0 +1,16 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
This file can be omitted as it doesn't contain anything.

Reply:
Removed